coderd/notifications/enqueuer.go

```diff
@@ -123,7 +123,8 @@ func (s *StoreEnqueuer) buildPayload(metadata database.FetchNewMessageMetadataRo
 	payload := types.MessagePayload{
 		Version: "1.0",
 
-		NotificationName: metadata.NotificationName,
+		NotificationName:       metadata.NotificationName,
+		NotificationTemplateID: metadata.NotificationTemplateID.String(),
```

Comment:
With the addition, we should bump the payload version by a minor release since it's purely additive.